### PR TITLE
Add Windows Documentation

### PR DESCRIPTION
**Pull request by ooeygui**

### CODE REVIEW

**Comment by ooeygui** [comment]
@dniewinski Let me know if you need any changes. I tried to minimize 'Microsoft messaging'. I wanted to minimize support burden by driving to our support channels; let me know if you'd like that messaging tweaked.

File: doc/winstartup.rst [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+**c:\catkin_ws\start_ros.bat**
+
+.. code:: batch
+    call c:\opt\ros\melodic\x64\setup.bat
+    call c:\catkin_ws\install\setup.bat
+    roslaunch <package> <launch file>
+
+Use the command Schtasks, to add a task which calls this script:
+.. code:: batch
+    schtasks /Create /RU <User> /RP <password> /SC ONLOGON /TN ROS /TR "c:\catkin_ws\start_ros.bat"
```
Comment (dniewinski) [inline doc/winstartup.rst:68]: You'll need an extra blank line under the code declaration

File: doc/winstartup.rst [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+To automatically start on ROS boot, you'll use the Windows Task Scheduler to start the task. The Task scheduler is 
+passed a Windows command file, which needs to start the ROS environment, and your code.
+
+Create a Windows command file, which includes the ROS environment and Install environment. 
+
+**c:\catkin_ws\start_ros.bat**
+
+.. code:: batch
+    call c:\opt\ros\melodic\x64\setup.bat
```
Comment (dniewinski) [inline doc/winstartup.rst:60]: You'll need an extra blank line under the code declaration

**Review by dniewinski — CHANGES_REQUESTED** [review]

File: doc/winstartup.rst (hunk 2 above)
Comment (ooeygui) [reply doc/winstartup.rst:60]: Fixed

File: doc/winstartup.rst (hunk 1 above)
Comment (ooeygui) [reply doc/winstartup.rst:68]: Fixed

File: doc/winsetup.rst [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+Once you've installed Windows and ROS, It's time to install the Jackal software. 
+
+.. code-block:: batch
+
+    mkdir c:\ws\src
+    cd c:\ws\src
+    git clone https://github.com/jackal/jackal_robot
+    git clone https://github.com/jackal/jackal
+    git clone https://github.com/jackal/jackal_firmware_ms
+    catkin_make
```
Comment (civerachb-cpr) [inline doc/winsetup.rst:54]: We're missing a cd .. command here; running catkin_make directly in the src directory fails.

**Review by civerachb-cpr — CHANGES_REQUESTED** [review]

**Comment by ooeygui** [comment]
Going to be iterating on this. I'll issue a new PR when done.

**Pull request closed without merging**